schema/release-schema.json

```diff
@@ -446,18 +446,36 @@
         "exclusionGrounds": {
           "title": "Exclusion grounds",
           "description": "The criteria regarding the situation of a tenderer that can lead to its exclusion from the contracting process. For example: criminal convictions, bankruptcy, presence on a blacklist or failure to pay taxes.",
-          "type": [
-            "string",
-            "null"
-          ]
+          "type": "object",
```

Comment:
The extension uses a `$ref` to a definition, for both these fields. Let's do the same for consistency.